tests/pytest/test.py

```diff
@@ -1000,7 +1000,8 @@ def testIssue_74(env):
 
     r.assertOk(r.execute_command('JSON.SET', 'test', '.', '{}'))
     # This shouldn't crash Redis
-    r.expect('JSON.SET', 'test', '$a', '12').raiseError()
+    r.expect('JSON.SET', 'test', '$a', '12').equal("OK")
```

Comment:
This is a progression (bug fix) - plain `12` is now parsed properly as numeric, which is good.
Seems we need to document this (in related Release Notes)